src/tools/CMakeLists.txt

```diff
@@ -7,7 +7,13 @@ set(rados_srcs
   ${PROJECT_SOURCE_DIR}/src/common/obj_bencher.cc
   ${PROJECT_SOURCE_DIR}/src/osd/ECUtil.cc)
 add_executable(rados ${rados_srcs})
-target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS} radosstriper)
+
+target_link_libraries(rados librados global ${BLKID_LIBRARIES} ${CMAKE_DL_LIBS})
+if(WITH_LIBRADOSSTRIPER)
+target_link_libraries(rados radosstriper)
```

Comment:
The indentation level and `(WITH_LIBRADOSSTRIPER)` text at the end of `else` and `endif` all match with the rest of the style of the rest of the file. I would think that if the indent level and `(...)` text after `else`/`endif` are to be changed, there should be a totally separate PR to change the entire file to match the new standard.

Reply:
not quite true. if you are referencing https://github.com/ceph/ceph/blob/master/src/tools/CMakeLists.txt#L13 , i'd say it's an exception. it's was introduced by a8f8598a73ec90fec719cc09e845eac755191939, where we wanted to avoid large code churn. because we were guarding a handful code blocks with `if(WITH_TESTS)` . but in this case, we're not changing existing code.

regarding to the `endif(FOO_BAR)` . i am trying to follow the coding convention of cmake upstream in recent changes.

i understand the coding style is always a controversial area. as you might know, Ceph enforces its own coding style, see https://github.com/ceph/ceph/blob/master/CodingStyle . there are multiple ways to handle it

1. claim that "the coding style is not enforced, because `src/foo/bar.cc` line 1234 does not follow it, and i am changing this file. so i'd like to use the existing code style in `bar.cc`".
1. try to follow the coding style in the new code or revised code. so we can "fix" it piecemeal
1. change the coding style en mass.

i'd prefer the 2nd option. because the net result of the first option is to "fork" the coding style in this project. and the 3rd one will be a disaster for backport and "git blame". one will take more effort to track down the change history of the change.

Reply:
@BlaineEXE I agree with @thaikov here that wholesale whitespace changes are a Bad Thing. They make it unnecessarily difficult to cherry-pick fixes from master to stable branches.

Reply:
@chardan Can we fix the whitespace piecemeal as @tchaikov is suggesting?

Reply:
@smithfarm Sure-- should be corrected in the current checkin.

Reply:
@chardan i am not sure if i've explain this already or not. but i will try again. i am proposing:
```cmake
if(WITH_LIBRADOSSTRIPER)
  target_link_libraries(rados radosstriper)
else()
  target_link_libraries(rados cls_lock_client)
endif()
```